MapboxNavigation.podspec

```diff
@@ -47,7 +47,6 @@ Pod::Spec.new do |s|
   s.dependency "MapboxMaps", "10.0.0-rc.3"
   s.dependency "Solar", "~> 2.1"
   s.dependency "MapboxSpeech-pre", "2.0.0-alpha.1"
-  s.dependency "MapboxGeocoder.swift", "~> 0.10.0"
```

Comment:
Ah, thanks for looking into this. Now I realized what you meant by removing `MapboxGeocoder`.